extension/src/popup/views/Swap/index.tsx

```diff
@@ -20,9 +20,7 @@ import { settingsNetworkDetailsSelector } from "popup/ducks/settings";
 
 export const Swap = () => {
```

Comment:
other misc bug found. When loading the swap page w/ no trustlines it gets caught in a useEffect loop.

I had fixed this before in https://github.com/stellar/freighter/pull/549, but looks like somewhere around the 2.3 merge conflict release it got added back 😬 . Hopefully with the new branch process we're doing won't be an issue!